packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.tsx

```diff
@@ -0,0 +1,161 @@
+                    </button>
+                </div>
+            </div>
+            <ActionSheet.Root
+                isOpen={is_open || is_open_description}
+                onClose={onActionSheetClose}
+                position='left'
+                className='accumulator-stats-sheet-wrapper'
+                expandable={false}
+            >
+                {is_open && (
+                    <ActionSheet.Portal shouldCloseOnDrag>
+                        <div className='stats-sheet'>
+                            <div className='stats-sheet__title'>
+                                <Text size='lg' bold>
+                                    <Localize i18n_default_text='Stats' />
+                                </Text>
+                            </div>
+                            <div className='stats-sheet__caption'>
+                                <Text size='md'>
+                                    <Localize i18n_default_text='History of tick counts' />
+                                </Text>
+                            </div>
+
+                            <div className='stats-sheet__stats'>
+                                <StatsRow
+                                    rows={rows[0]}
+                                    animationClass={animationClass}
+                                    isMovingTransition={isMovingTransition}
+                                    className='stats-sheet__stats'
+                                />
+                            </div>
+                            {rows.slice(1).map((item, index) => (
+                                <div key={index} className='stats-sheet__stats'>
+                                    {item.map((item: number, innerIndex: number) => (
+                                        <div key={`${index}-${innerIndex}`} className='stats-sheet__stats__number'>
+                                            <Text size='sm'>{item}</Text>
+                                        </div>
+                                    ))}
+                                </div>
+                            ))}
+                        </div>
+                    </ActionSheet.Portal>
+                )}
+                {is_open_description && (
+                    <ActionSheet.Portal showHandlebar={false}>
+                        <div className='stats-description'>
+                            <div className='stats-description__placeholder' />
+                            <div className='stats-description__content'>
+                                <div className='stats-description__content__title'>
+                                    <Text size='lg' bold>
+                                        <Localize i18n_default_text='Stats' />
+                                    </Text>
+                                </div>
+                                <div className='stats-description__content__description'>
+                                    <Text size='md'>
```

Comment:
```suggestion
                                    <Text>
```
nit-pick: `size='md'` is the default one )